static/js/zamboni/reviewers.js

```diff
@@ -222,7 +222,7 @@ function initExtraReviewActions() {
     }));
 
     // One-off-style buttons.
-    $('#clear_admin_code_review, #clear_admin_content_review, #clear_pending_info_request, #clear_auto_approval_delayed_until').click(_pd(function() {
+    $('#clear_admin_code_review, #clear_admin_content_review, #clear_admin_theme_review, #clear_pending_info_request, #clear_auto_approval_delayed_until').click(_pd(function() {
```

Comment:
ugh, this is not a good coding pattern.  They should have a common class instead.

Reply:
Yeah that one is my fault, it started with just a couple buttons and grew...